recursive/methods/build.rs

```diff
@@ -1,3 +1,13 @@
+use std::collections::HashMap;
+
 fn main() {
-    risc0_build::embed_methods();
+    let options_map = HashMap::from([(
+        "methods-guest",
+        risc0_build::GuestOptions {
+            // change to 16 to run miden fully (will increase compile time)
+            code_limit: 12,
```

Comment:
@maxgillett this needs to be changed to 16 to finish verification successfully (but it really lengthens compile time). I put it at 12 since it doesn't really matter if the program verifies for our optimizing cycle